app/src/main/java/com/github/se/polyfit/ui/components/AddIngredientPopup.kt

```diff
@@ -137,19 +162,26 @@ fun SearchIngredients() {
       }
 }
 
-// @Composable
-// fun AddButton(onClick: () -> Unit, modifier: Modifier) {
-//  Button(
-//      onClick = onClick,
-//      modifier = modifier.padding(top = 16.dp).fillMaxWidth(0.5f),
-//      shape = RoundedCornerShape(50.dp),
-//      colors = ButtonDefaults.buttonColors(containerColor = PrimaryPurple)) {
-//        Text("Add", color = Color.White)
-//      }
-// }
-
 @Composable
-fun AddIngredientDialog(onClickCloseDialog: () -> Unit, onAddIngredient: (Ingredient) -> Unit) {
```

Comment:
since this is the main component of the file, leave this at the top, to follow with how we do in others

Reply:
done